indigo_app/templates/indigo_api/work_amendments.html

```diff
@@ -6,6 +6,13 @@
 {% block work-content %}
   <h4>Points in time</h4>
 
+  {% if work.stub %}
+    <p class="alert alert-warning">
+      <i class="fas fa-exclamation-triangle text-warning"></i> Stub works should not have points in time.
+      {% if work.amendments.exists %}Either remove these points in time or don't mark this work as a stub.{% endif %}
+    </p>
+  {% endif %}
+
```

Comment:
should a similar warning also appear on the work overview if there are docs?

Reply:
Added.